app/src/main/java/com/atomicrobot/carbon/monitoring/CrashlyticsCrashReporter.kt

```diff
@@ -0,0 +1,13 @@
+package com.atomicrobot.carbon.monitoring
+
+import com.google.firebase.crashlytics.FirebaseCrashlytics
+
+class CrashlyticsCrashReporter : CrashReporter {
+
+    override fun logMessage(message: String) = FirebaseCrashlytics.getInstance().log(message)
+
+    override fun logException(message: String, ex: Exception) {
+        FirebaseCrashlytics.getInstance().log(message)
+        FirebaseCrashlytics.getInstance().recordException(ex)
+    }
+}
```

Comment:
This is just a general comment to think about in the future - might be worthwhile to hide this behind a timber tree so that logs will automatically be logged both to the console and to firebase instead of needing to potentially add both logs.

Reply:
Lets create a ClickUp ticket for this one and do it later